resources/views/users/meetings/modals/calendar-action.blade.php

```diff
@@ -58,6 +58,25 @@
             </div>
         </div>
     </div>
+    <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+    <script>
+        $(document).ready(function () {
+            $('.js-count-text').on('input', function () {
+                const text = $(this).val();
+                const errorElement = $('#error-message');
+
+                if (text.length > 50) {
```

Comment:
does it necessary to limit the character into 50?

Reply:
Because it's title of meeting, so we don't want to be too long.
Should we change the limit 50 to 100?